p4-16/spec/P4-16-spec.mdk

```diff
@@ -4946,21 +4946,20 @@ switchCase
 
 switchLabel
     : name
+    | expression
```

Comment:
I am trying to implement this, but the grammar becomes ambiguous, because now if you see a { after a colon you no longer know whether it's a list expression, part of the next label, or a block statement. So we may need to add some other disambiguating factor, like a 'case' label.